di/src/main/java/com/interface21/beans/factory/config/BeanDefinitionImpl.java

```diff
@@ -0,0 +1,21 @@
+package com.interface21.beans.factory.config;
+
+public class BeanDefinitionImpl implements BeanDefinition {
```

Comment:
```suggestion
public class DefaultBeanDefinition implements BeanDefinition {
```

impl이라는 이름보다 구현체를 구분할 수 있는 이름을 붙이면 어떨까요?
MethodBeanDefinitionImpl은 impl을 지우더라도 이름에서 메소드로 빈을 정의하는 것을 알 수 있는 것처럼 적절한 이름을 붙이면 좀 더 코드 파악에 용이하겠네요.